src/os/bluestore/BlueStore.cc

```diff
@@ -12041,6 +12097,8 @@ void BlueStore::collect_metadata(map<string,string> *pm)
   }
   (*pm)["bluestore_min_alloc_size"] = stringify(min_alloc_size);
   (*pm)["bluestore_allocation_from_file"] = stringify(fm && fm->is_null_manager());
+  (*pm)["bluestore_write_mode"] = use_write_v2 ? "new" : "classic";
+  (*pm)["bluestore_onode_segmentation"] = segment_size == 0 ? "inactive" : "active";
```

Comment:
why not reporting segment size as-is? It's more helpful IMO.

Reply:
what's about this one?

Reply:
I thought about printing size here, but segment_size might  differ for different pools.
I preferred to be imprecise than wrong.

Reply:
Well, agree with this statement if global segment_size would control segmentation for any onode, including ones with persistent non-zero segment size. ;)